src/mcts/node.h

```diff
@@ -345,12 +345,10 @@ class EdgeAndNode {
   Node* node() const { return node_; }
 
   // Proxy functions for easier access to node/edge.
-  float GetQ(float default_q, float draw_score, bool logit_q) const {
+  float GetQ(float default_q, float draw_score, bool logit_q, float scale = 1.0f) const {
```

Comment:
Instead of carrying two variables everywhere, I think it's better to have one `std::optional<float>`.

Also I believe in this case having default function arguments do more harm than good, it's easy to confuse parameters.